easybuild/easyconfigs/s/SAMtools/SAMtools-1.8-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Robert Schmidt <roschmidt@ohri.ca>, Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+#
+# Modified for version 1.4 by: Kurt Lust, UAntwerpen
+#
+##
+name = 'SAMtools'
+version = "1.8"
+
+homepage = 'http://www.htslib.org/'
+
+description = """
+ SAM Tools provide various utilities for manipulating alignments in the SAM
+ format, including sorting, merging, indexing and generating alignments in
+ a per-position format.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```

Comment:
We usually use a full toolchain like `intel` or `foss` here, for performance reasons (building with Intel compilers rather than GCC), same question as in #6207

Thoughts on this @verdurin, @vanzod?

Reply:
I would not pull it down to `GCCcore`. `SAMtools` may benefit from being built with one compiler versus another. @JackPerdue, is there a specific reason you need it built with `GCCcore`?

Reply:
ping @JackPerdue?